projects/common/src/lib/util/directives/flexbox-legacy.directive.ts

```diff
@@ -61,7 +61,7 @@ export class FxLayoutDirective implements OnChanges {
     selector: '[fxLayoutGap]',
 })
 export class FxLayoutGapDirective implements OnChanges {
-    @Input() fxLayoutGap: '0.5rem' | '1rem' = '1rem';
+    @Input() fxLayoutGap: '0.5rem' | '1rem' | '8px' = '1rem';
```

Comment:
just use 0.5rem instead of 8px

Reply:
928a14211d2671f7a02ef4a89348644421b054ab